### PR TITLE
Add state prefix how-to

### PR DESCRIPTION
**Pull request by yaron2**
This PR adds information for a new feature that enables developers to set a key prefix for a state store.

### CODE REVIEW

File: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state copy.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+type: docs
+title: "How-To: Change a state sharing strategy"
+linkTitle: "How-To: Change a state sharing strategy"
+weight: 400
+description: "Choose a strategy for sharing state between different applications"
+---
+
+## Introduction
+
+Dapr offers developers differnet ways to share state between applications.
```
Comment (vinayada1) [inline daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state copy.md]: typo: different

File: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state copy.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
+---
+type: docs
+title: "How-To: Change a state sharing strategy"
+linkTitle: "How-To: Change a state sharing strategy"
+weight: 400
```
Comment (vinayada1) [inline daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state copy.md]: Why does this file have "copy" in the name?

File: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state copy.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+description: "Choose a strategy for sharing state between different applications"
+---
+
+## Introduction
+
+Dapr offers developers differnet ways to share state between applications.
+
+Different architectures might have different needs when it comes to sharing state. For example, in one scenario you many want to encapsulate all state within a given application and have Dapr manage the access for you. In a different scenario, you may need to have two applications working on the same state be able to get and save the same keys.
```
Comment (vinayada1) [inline daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state copy.md]: many -> may

File: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state copy.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+To enable state sharing, Dapr supports the following key prefixes settings:
```
Comment (vinayada1) [inline daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state copy.md]: settings -> strategies?

File: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state copy.md (hunk 2 above)
Comment (AaronCrawfis) [reply daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state copy.md]: Good catch, this should be changed to `howto-change-state-sharing.md`

File: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state copy.md (hunk 4 above)
Comment (yaron2) [reply daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state copy.md]: changed

File: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state copy.md (hunk 3 above)
Comment (yaron2) [reply daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state copy.md]: changed

File: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state copy.md (hunk 2 above)
Comment (yaron2) [reply daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state copy.md]: changed

File: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state copy.md (hunk 1 above)
Comment (yaron2) [reply daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state copy.md]: fixed

**Review by vinayada1 — APPROVED** [review]

File: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-share-state.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
+---
+type: docs
+title: "How-To: Share state between applications"
+linkTitle: "How-To: CShare state between applications"
```
Comment (AaronCrawfis) [inline daprdocs/content/en/developing-applications/building-blocks/state-management/howto-share-state.md]: type: Share

**Review by AaronCrawfis — APPROVED** [review]

File: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-share-state.md (hunk 5 above)
Comment (yaron2) [reply daprdocs/content/en/developing-applications/building-blocks/state-management/howto-share-state.md]: Fixed

**Pull request merged**